source/replication/enable-server-side-two-way-bucket-replication.rst

```diff
@@ -0,0 +1,514 @@
+
+Requirements
+------------
+
+Enable Versioning on Source and Destination Buckets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO relies on the immutability protections provided by versioning to
+synchronize objects between the source and replication target.
+
+Use the :mc-cmd:`mc version enable` command to enable versioning on 
+*both* the source and destination bucket before starting this procedure:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc version enable ALIAS/PATH
+
+- Replace :mc-cmd:`ALIAS <mc version enable TARGET>` with the
+  :mc:`alias <mc alias>` of the MinIO cluster.
+
+- Replace :mc-cmd:`PATH <mc version enable TARGET>` with the bucket on which
+  to enable versioning.
+
+Install and Configure ``mc`` with Access to Both Clusters.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This procedure uses :mc:`mc` for performing operations on both the source and
+destination MinIO cluster. Install :mc:`mc` on a machine with network access to
+both source and destination clusters. See the ``mc`` 
+:ref:`Installation Quickstart <mc-install>` for instructions on downloading and
+installing ``mc``.
+
+.. _minio-bucket-replication-serverside-twoway-permissions:
+
+Required Permissions
+~~~~~~~~~~~~~~~~~~~~
+
+Bucket replication requires specific permissions on the source and
+destination clusters to configure and enable replication rules. 
+
+.. tabs::
+
+   .. tab:: Replication Admin
+
+      The following policy provides permissions for configuring and enabling
+      replication on a cluster.
+
+      .. code-block:: shell
+         :class: copyable
+
+         {
+            "Version": "2012-10-17",
+            "Statement": [
+               {
+                     "Action": [
+                        "admin:SetBucketTarget",
+                        "admin:GetBucketTarget"
+                     ],
+                     "Effect": "Allow",
+                     "Sid": "EnableRemoteBucketConfiguration"
+               },
+               {
+                     "Effect": "Allow",
+                     "Action": [
+                        "s3:GetReplicationConfiguration",
+                        "s3:ListBucket",
+                        "s3:ListBucketMultipartUploads",
+                        "s3:GetBucketLocation",
+                        "s3:GetBucketVersioning"
+                     ],
+                     "Resource": [
+                        "arn:aws:s3:::*"
+                     ],
+                     "Sid": "EnableReplicationRuleConfiguration"
+               }
+            ]
+         }
+
+      The ``"arn:aws:s3:::*`` resource applies the replication permissions to
+      *any* bucket on the source cluster. You can restrict the user policy to
+      specific buckets as-needed.
+
+      Use the :mc-cmd:`mc admin policy set` command to associate the policy to
+      a user on the source MinIO cluster.
+
+   .. tab:: Replication Remote Admin
```

Comment:
would prefer to see this `Replication Remote Admin` referred to as a user rather than admin. w.r.t replication, the admin needs permissions to configure the remote targets, while the replication user just needs permissions to receive replicated content.